PerformanceTests/JetStream2/JetStreamDriver.js

```diff
@@ -494,7 +496,7 @@ class Driver {
         if (!isInBrowser)
             return;
 
-        if (window.location.search !== '?report=true')
+        if (!urlParameters.has('report'))
```

Comment:
I think we should keep using `urlParameters.get('report')` as `?report=false` is a valid case and in that case we want the early return in next line.

Reply:
Also,  `urlParameters.get('report')` gives `"false"` string, which will be evaluated true.
If we only allow `report` key to have "true" or "false" as input, then, we should do
`if (urlParameters.get('report') != 'true')` instead.